bitbots_quintic_walk/include/bitbots_quintic_walk/walk_node.h

```diff
@@ -54,19 +55,24 @@ namespace bitbots_quintic_walk {
 
 class WalkNode {
  public:
-  WalkNode();
+  WalkNode(const std::string ns);
+  bitbots_msgs::JointCommand step(double dt);
   bitbots_msgs::JointCommand step(
       double dt,
       const geometry_msgs::Twist &cmdvel_msg,
       const sensor_msgs::Imu &imu_msg,
-      const sensor_msgs::JointState &jointstate_msg);
+      const sensor_msgs::JointState &jointstate_msg,
+      const bitbots_msgs::FootPressure &pressure_left,
+      const bitbots_msgs::FootPressure &pressure_right);
   /**
    * Small helper method to get foot position via python wrapper
    */
   geometry_msgs::Pose get_left_foot_pose();
 
   void reset();
 
+  void specialReset(WalkState state, double phase, geometry_msgs::Twist cmd_vel, bool reset_odometry);
```

Comment:
Add a docstring about what this `specialReset` does and optionally rename to `reset`.